css/style.css

```diff
@@ -0,0 +1,140 @@
+*{  padding: 0; margin: 0;  border: 0;}
+*,*:before,*:after{ -moz-box-sizing: border-box;  -webkit-box-sizing: border-box; box-sizing: border-box;}
+:focus,:active{outline: none;}
+a:focus,a:active{outline: none;}
+nav,footer,header,aside{display: block;}
+html,body{height: 100%; width: 100%;font-size: 100%;line-height: 1;font-size: 14px;-ms-text-size-adjust: 100%;-moz-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;}
+input,button,textarea{font-family:inherit;}
+input::-ms-clear{display: none;}
+button{cursor: pointer;}
+button::-moz-focus-inner {padding:0;border:0;}
+a, a:visited{text-decoration: none;}
+a:hover{text-decoration: none;}
+ul li{list-style: none;}
+img{vertical-align: top;}
+h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight: 400;}
+
+
+
+.wrapper {
+  background-color: #e0e0e0;
+  font-family: 'Open Sans Condensed', sans-serif;
+  color: #afafad;
+  list-style-type: none;
+  overflow: hidden;
+  min-height: 100%;
+}
+._container {
+  max-width: 400px;
+  margin: 0 auto;
+}
+.page {
+  padding: 50px 0;
+}
+.page__title {
+  font-size: 100px;
+  text-align: center;
+  margin: 0 0 20px 0;
+  color: #f6dbac;
+}
+.page__body {
+  background-color: #ffffff;
+  border: 2px solid #afafad;
+  border-bottom: 1px solid #afafad;
+  box-shadow: 5px 5px 5px #afafad;
+  border-radius: 5px;
+}
+.body {}
+.body__add-task {
+  padding: 0 0 0 30px;

```

Comment:
from design points of view all margin and padding should be aliquot to 8PX or 0,5REM